pkgs/development/python-modules/numba/default.nix

```
@@ -16,21 +16,33 @@ buildPythonPackage rec {
   version = "0.45.0";
   pname = "numba";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "fcea8dc6f9e0f1ddf7bd52a207858539bc14e893c5ee66d8730c3e5b9344c4b3";
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = version;
+    sha256 = "1kb6jqwad8wy5nnk476g1vfbnyqv6lafz3mcyiwxdsi14hwps902";
   };
 
   NIX_CFLAGS_COMPILE = stdenv.lib.optionalString stdenv.isDarwin "-I${libcxx}/include/c++/v1";
 
-  propagatedBuildInputs = [numpy llvmlite] ++ stdenv.lib.optional (!isPy3k) funcsigs ++ stdenv.lib.optional (isPy27 || isPy33) singledispatch;
+  propagatedBuildInputs = [
+    numpy
+    llvmlite
+  ] ++ stdenv.lib.optional (!isPy3k) funcsigs
```

python3.3 is not available in nixpkgs, should only need to check python2.7 vs python3.5+
```suggestion
  ] ++ stdenv.lib.optionals (!isPy3k) [ funcsigs singledispatch ];
```